README.md

```diff
@@ -79,7 +78,8 @@ Want to quickly make a trading app? Here you go:
     
         # Handle errors.
         def error_callback(self, error):
-            print("Errors:", json.dumps(error, indent=2))
+            print("Add your error message.")
+            #print("Errors:", json.dumps(error, indent=2))
```

Comment:
I think I preferred the way it was before, the comment in line 79 is explanatory enough in my view.